### ISSUE TITLE
Setting textColor with mode prop set as date/datetime breaks to default text color

### ISSUE THREAD

**Issue by ptboyer** [henninghall/react-native-date-picker#119]
**the bug**
Setting `textColor` with `mode` prop set to `"date"` or `"datetime"` will incorrectly display "TODAY" as the default black color for light/white backgrounds, ignoring the `textColor`.

| props.mode   | props.textColor | Result                                                       |
| ------------ | --------------- | ------------------------------------------------------------ |
| not set      | `"#FFFFFF"`     | ![image](https://user-images.githubusercontent.com/8391902/67255147-62d2d200-f4cc-11e9-8bda-6391641063bb.png) |
| `"datetime"` | `"#FFFFFF"`     | ![image](https://user-images.githubusercontent.com/8391902/67255156-741bde80-f4cc-11e9-8692-dc26b868d64b.png) |
| `"date"`     | `"#FFFFFF"`     | ![image](https://user-images.githubusercontent.com/8391902/67255163-7aaa5600-f4cc-11e9-84ab-759690585090.png) |
| `"time"`     | `"#FFFFFF"`     | ![image](https://user-images.githubusercontent.com/8391902/67255176-8564eb00-f4cc-11e9-8170-946132aa8f22.png) |

**Expected behavior**
Setting `textColor` with `mode` prop set to `"date"` or `"datetime"` will correctly set the current "Today" colour tint for the date picker component.

**To Reproduce**
Add example code that reproduces the behavior. 

```javascript
export default class App extends Component {
  state = { date: new Date() }
  render = () =>
    <DatePicker
      date={this.state.date}
      onDateChange={date => this.setState({ date })}
      mode="datetime"
      textColor="#FFFFFF"
    />
}
```

**Smartphone (please complete the following information):**
 - OS: iOS
 - React Native version 0.61.2
 - react-native-date-picker version 2.7.0

**Comment by ptboyer** [comment]
Fixed it, made a pull request with change, as suggested by [link](https://stackoverflow.com/questions/20875054/change-uidatepicker-font-color#comment65147552_31775498).

**Comment by r4mdat** [comment]
Also ran into this issue, PR works like a charm. Would be great to get it merged. Great library.

**Comment by cspicuzza** [comment]
@ptboyer thanks for those prs!   @henninghall Can we get those merged?

**Comment by henninghall** [comment]
Released a fix for this in `2.7.3`
